.prospector.yaml

```diff
@@ -1,24 +1,32 @@
-strictness: medium     # default strictness
-test-warnings: false   # inspect unittests
-doc-warnings: true     # inspect docstrings
+strictness: medium       # default strictness
+test-warnings: false     # inspect unittests
+doc-warnings: true       # inspect docstrings
 
-ignore-paths:          # ignore the docs folder
+ignore-paths:            # ignore the docs folder
   - docs
   - ci_scripts
   - examples
 
 pep8:
   disable:
-    - N806              # ignore pep8 naming (func arg should be lowercase i.e. X_pred)
+    - N802               # ignore pep8 naming (func name should be lowercase i.e. get_X_y())
+    - N806               # ignore pep8 naming (func arg should be lowercase i.e. X_pred)
   options:
     max-line-length: 120
 
+pep257:
+  disable:
+    - D203               # 1 blank line required before class docstring (conflicts with D211 [no blanks allowed])
+    - D212               # Multi-line docstring summary should start at the first line(conflicts with D213 [first line])
+    - D205               # 1 blank line required between summary line and description
```

Comment:
I suggest to leave that in.

Reply:
Well we would just have to decide if we want to use D211 or D203 and D212 or D213.
I just had to choose one of the conflicting rules.